clients/@grouparoo/web/README.md

```diff
@@ -0,0 +1,121 @@
+# @grouparoo/client-web
+
+Grouparoo provides a javascript library for tracking events from browsers and sending them to your Grouparoo server.
+
+## Install
+
+You can include this library in a few ways:
+
+**Javascript / Typescript Module**
+
+For use in environments that will be compiled, ie: React or Webpack.
+
+```ts
+import { GrouparooWebClient } from "@grouparoo/client-web";
+const client = new GrouparooWebClient({
+  host: "https://grouparoo.my-company.com",
+  apiKey: "abc123zzz",
+});
+```
+
+**Script Tag**
+
+```html
+<html>
+  <head>
+    <script
+      type="application/javascript"
+      href="https://grouparoo.my-company.com/public/javascript/grouparooWebCleint.js"
+    />
+  </head>
+
+  <script>
+    const client = new GrouparooWebClient({
+      host: "https://grouparoo.my-company.com",
+      apiKey: "abc123zzz",
+    });
+  </script>
+</html>
+```
+
+To initialize a client, you'll need to provide:
+
+- **host** (required): The publicly-available URL of your Grouparoo server/cluster.
+- **apiKey** (required): An API Key that you have generated in your Grouparoo server, with `write` access to the `events` topic.
+- **path** (optional): The route to post events too. Default: `/api/v1/track`
+- **anonymousId** (optional): An ID you would like to use for the `anonymousId` for this client. By default, we will generate a unique UUID for this browser and store it in a cookie for future use.
```

Comment:
is this a boolean? I can't figure out how it would make sense to give an actual value, but the words suggest you could pass an ID.

Reply:
maybe move this one down in the doc to buy the cookie and cookieTTL as they are all the same. I wonder if it should be one setting actually
```
anonymousId: false
```
or 
```
anonymousId: true
```
or
```
anonymousId: { cookie: "grouparoo-anonymous-id", ttl: 90*24*60*60 }
```

Reply:
Actually, they aren't the same:

Say your app has your own notion of `anonymousId`, (like TR) that might be hydrated into the page, from a cookie, in the URL (J_sessionId), or something else.  This allows you to pull that in directly and not use our random UUID thing.    Once you set it, the Grouparoo client will store that in our cookie, but it doesn't need to originate there.

... I'll add this all to the readme!